docs/source/implementations/feature_importance.rst

```diff
@@ -58,16 +59,22 @@ An example showing how to use various feature importance functions:
 
   oos_score = ml_cross_val_score(clf, X_train, y_train, cv_gen=cv_gen, sample_weight_train=None, scoring=accuracy_score).mean()
 
+  feature_clusters = get_feature_clusters(X_train, dependence_metric='information_variation', distance_metric='angular',
+                                          linkage_method='single', n_clusters=None)
+
   mdi_feature_imp = mean_decrease_impurity(clf, X_train.columns)
   mda_feature_imp = mean_decrease_accuracy(clf, X_train, y_train, cv_gen, scoring=log_loss)
   sfi_feature_imp = single_feature_importance(clf, X_train, y_train, cv_gen, scoring=accuracy_score)
+  cfi_feature_imp = mean_decrease_accuracy(clf, X_train, y_train, cv_gen, clustered_subsets=feature_clusters, scoring=log_loss)
 
   plot_feature_importance(mdi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='mdi_feat_imp.png')
   plot_feature_importance(mda_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='mda_feat_imp.png')
   plot_feature_importance(sfi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='sfi_feat_imp.png')
+  plot_feature_importance(cfi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
+                                save_fig=True, output_path='cfi_feat_imp.png')
 
 The following are the resulting images from the MDI, MDA, and SFI feature importances respectively:
```

Comment:
Please add a tutorial notebook to the research repo and link back to here, on how to use CFI.

Reply:
I was thinking to add that after the release of CFI. And I think it requires a different notebook and will not be a part of chapter-8 notebook tough it is a feature importance algorithm.

Reply:
I agree, lets do a seperate notebook that explores the concept. I am able to add notebooks as tutorials to the docs. Thats what I am currently working on. We need to have amazing documentation, its the number one feedback from our users. 

However lets do it before we do the release because I want to make sure we don't need to make changes again.

You can use the code by executing: `pip install -e .` this will create a symlink to your local copy of the package, then you don't need to wait for a version release.

Reply:
Thanks for the help. I will write the notebook on CFI.